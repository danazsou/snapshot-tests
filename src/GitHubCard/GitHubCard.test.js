import React from "react"
import Card from "react-bootstrap/Card"


function GitHubCard() {
  return (
      <div className="card">
        <Card style={{ width: "25rem"}}>
        <Card.Img variant="top" src="https://avatars.githubusercontent.com/u/106777410?s=96&v=4" />
        <Card.Body>
            <Card.Title>Dana</Card.Title>
            <Card.Text>
            Hi - my name is Dana. I am currently studying software development. 
            it has but extremely challenging but also fun.
            </Card.Text>
        </Card.Body>
        </Card>
      </div>
  );
}

export default GitHubCard