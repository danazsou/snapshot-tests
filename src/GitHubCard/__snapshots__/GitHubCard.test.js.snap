// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <div
    className="card"
    style={
      Object {
        "width": "25rem",
      }
    }
  >
    <img
      className="card-img-top"
      src="https://avatars.githubusercontent.com/u/106777410?s=96&v=4"
    />
    <div
      className="card-body"
    >
      <div
        className="card-title h5"
      >
        Dana
      </div>
      <p
        className="card-text"
      >
        Hi - my name is Dana. I am currently studying software development. it has but extremely challenging but also fun.
      </p>
    </div>
  </div>
</div>
`;
